examples/cfd/test_04_burgers.ipynb

```diff
@@ -0,0 +1,319 @@
FD8FGFomuYpmucWoTvyyCNj\nf06DwpCcFuPFqUFvFCfOpESOkbYgGYaBdruNRqOBbDbrKHKy0xkVp1ch4Z3mGdPA19Ndg5ctW8A5\n+T2uVhayRqGSYtCPjdeWKnZRZfYBxPoaoi1XfwzelVIAWKVrGPHiRa5QKCRWxZmW0PEil8lkUK1W\nzRC/G2kL6MCgiHcMn902B8UH49jg9E/QwoikCNJOxW8UhQjGMJ+fvEbz2I5Mq9XC1q1bccMNN+Do\no4/Grl27BvIDfVzQkYqAKOe5Wqs4k27HkbQgGYZhzlsFgGq1ilwu5/tCMjTRgSF4ioFIsBedTPgp\nwPDSmHYQxkylBR23vVijeWyyj6IoGB0dxRFHHIENGzbgscceww9+8APsv//+WLJkCY499lgce+yx\nOOecc/pOkLjkkkuwZs0azJkzBxs3buz5+V133YVrrrkGiqJgZGQE3/jGN7BkyZJYnm9SkNDFQBAp\nSlvkGEkJHWsIXK/XAQDlchn5fN73SU/EQg6CEJmg0TynMVMkee4Mc4TOK+z8rSgK5syZg8suuwwA\n8I53vAO//vWv8frrr2Pjxo34zW9+g3vuuQfnnHNO3/tcsWIFPvaxj2H58uW2P1+4cCH++7//GzNm\nzMB9992HSy+9FOvXr4/0eSUNCV0M+JEiq8hZqziTJgmhYxE5XddRLpdRKBRCXRBEu5jQyK9oGPnx\nKDojaa9ieAgSzeNHnbH3YbvdHopRZ36g4+CM2/Wm1WqhUqng8MMPx+GHH47zzjvP8/0uW7YMmzZt\ncvz5qaee2vXvLVu2eL5vUSGhiwCnogg3RBM5RpxCp6oqGo0GNE2LROQYIuXRdTqdtJdA7CGuXnTb\nPn465nz9FzHcs3j4ieZ1Oh0z4Z1Gne1FlHOT6FhfE0nOBv7Wt76Ft7zlLbE/TtyQ0MWAm2CIKnKM\nOORI0zTU63UzT2JkZCTSN6kIQse2j/mxb6MvA1P/x/99HfrdvVL44gX9C0Nixwj3f1V+rc/r20Pr\nkjSx9qIj7EXvezv+FZlMxsxtolFnexnk5xaWfukycR+7hx56CKtXr8YjjzwS6+MkAQldBNhF6Kzz\nXHVdR6vVcmyQKwpRypF1vmzUIicCVlktFou+fn/hD1q93+SOES93drx4oQDClwJp9KITvdI1bd6z\nz0rb7w/7qLO0P2yKjpPQJXHcNm7ciJUrV+K+++4biPYoJHQRwYtQJpOBpmkA+k86EA07GfULP82i\nVCqhWq3GelJOI0LXT1ZHX3b/fVuRC8Ch9/QK3yBI3siPRwEA+WkDnRG5L+jDzqCOOvODbOtNEieh\nazQaqFQqoe/b6drwyiuv4LzzzsN3vvMdHHrooaEeRxRI6GKASVGj0ZBG5Bhh5CjpJsiMJIWO3zIP\n+hwPXlODXojvuNhJHjAYoudKQq1LRGguLDtOkmfXr0z2UWcUoQvG+Pg4Zs6cGfj3L7roIqxbtw47\nduzAQQcdhKuuugrtdhuKomDlypX40pe+hJ07d+KjH/0oDMNAPp/HE088EeEzSB4SuojgZ7h2Oh2o\nqopMJiONyDFkbbkS90mTj7Q6Pce3ld+HTf9wuvm1Xf7cwWtqsa7TjUGN5hGDQZjmyKKPOhNhDaLi\nFKELK3R33XWX689vvfVW3HrrrYHvX0RI6CLCMAwzIsdOKiMj8vVc8CN0VslJS17jPFnyM2WdIq1v\nK7/P/LfbVmuaMueEreS918dpgSaYETEj6qgzr1CEzp24hG4YIaGLiOnpaei6jtHRUSiKgqmpqbSX\nFAgvQidaXmAcW65eZsryItcPEWXOiUPvVm2/H2s0j6t0ZflzIkKFEWIg26gzitA54yR0ExMTJHQ+\nIaGLiNHRUbOYgJ08ZMRNjvhoVS6Xs5WcNIhS6PhRZIqi2M6U9SNyg4Q1mifDdi31ohsuRBx1RhG6\nYJDQ+Sf9q/GAwL/pZR9HZT0BeYlWDQJM5AzDsB1FFlTkZIrO+eHQezrAntfKixcUEn/8JFuXUC86\neYlr1Jmfc7uM14GkYCJtZWJiAgsWLEhhRfIyeFfllLAKHYsayfZG5tfLi1wmk7GNVolA2AhdvwkW\nQUVu9GVg9m8HU+asHPrdNoB4xI5alxBxEHbUmVXynLZsZbwOJInbluuSJUtSWJG8kNDFhAjTC4LA\n1s2aIANAtVpFLpcT9qQUtHce30vOaYKFH5nb+aHT+99owAkrdiLnzzFKj8pX7ER4I0g0T9M011Fn\nJHTuOF0nacvVPyR0ERFknqtoGIYBVd2dEN9sNm23HUXFz7Hm++U5NT6OIk/OKTrn2oNOgmPtBSZ2\nr/1/Ze+/9KsxhM54S6gXHTFcuEXz2K6F26gz9r1hGXXmFyqKiAYSupiQTehY/hiLdI2MjEjTP8/r\nidFLU+BhLXggCMIfbs2R+Rzjer1ufj0Mo8784rblOgjjuJKEhC4mohihlQR2+WMTExNpL8sX/eTZ\nS5sVEjmCIMLitGX7vR3/2jPPdpBHnfmB2pZEBwldRFhfkCx3QlSYyLGh8nz+mGzRRaf1WtusJCly\n+z22Hdqoj+1GQjom/+o0jN35WNrLICTgPfus7PneoI4684PbdUZVVRQKyVfPywwJXUyIKkX9hsoD\n4q7dK9bq3LBNgYngHPSTcbzy1nQ/ZcfVi66xr4KxGO6XGA4GedSZX6xrlvn6kyYkdBEhuhRpmoZm\ns+laCMAQbe394OfoqqqKer0OAIk2BeYrXPd7bHssjzHoHLC+hakF9p/InVqXUC86YtCQfdSZH/pV\nAIu4ZpEhoYsQXoREyaHjKzqdCgGsyCh0uq5jamoKuq6jUqlE1hSYIAgibWQbdeYVJ6GjVi/BIKGL\nibSliK/oLBQKnkSOkfba/aBpGur1OgzDQKFQQLFYjKUFie91Uf6ciZ9t19HNbcconYhQHh2RJiKO\nOouCqakpjIx
```

Comment:
I' assuming you are talking about the explicit `first_derivative` derivation? If so, yes, it it. I think it might also be nice to demonstrate operators with mixed spatial orders, in case people might want this.